### PR TITLE
Update Signin.js

### PR DESCRIPTION
**Pull request by Eunit99**
* Transformed Material UI into Bootstrap CSS

### CODE REVIEW

**Review by leanny01 — CHANGES_REQUESTED** [review]
This is a good start, please my comment bellow:

- Often all style files, images and assets are placed in a folder called assets other styles are placed right under existing  components  themselves, please see this tutorial:

https://www.pluralsight.com/guides/file-structure-react-applications-created-create-react-app

- Also the `./Animate.min.css` doesn't seem to load, please have a look

**Comment by Eunit99** [comment]
Noted, will make the necessary corrections.

File: src/components/core/Sidenav.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { faDashboard, faBriefcase, faUser, faUsers, faCogs, faLegal } from '@fortawesome/freesolid-svg-icons';
+
+class Sidenav extends Component {
+    render() {
+        const FaDashboard = <FontAwesome icon={["fal", "dashboard"]} />,
+            FaBriefcase = <FontAwesome icon={["fal", "briefcase"]} /> ,
+            FaUser = <FontAwesome icon={["fal", "user"]} />,
+            FaUsers = <FontAwesome icon={["fal", "users"]} />,
+            FaCogs = <FontAwesome icon={["fal", "cogs"]} />,
+            FaLegal = <FontAwesome icon={["fal", "legal"]} />;
```
Comment (leanny01) [inline src/components/core/Sidenav.js]: There seems to be a problem with how you have declared the Fontawesome and how you use it, it is preventing the app from from running please check it out.

File: src/components/core/Sidenav.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```
Comment (leanny01) [inline src/components/core/Sidenav.js]: Please check to see how it is implemented, there seems to be a problem.

**Review by leanny01 — CHANGES_REQUESTED** [review]
Please see my comments.

File: src/components/core/Sidenav.js (hunk 2 above)
Comment (Eunit99) [reply src/components/core/Sidenav.js]: Working on them.

**Review by leanny01 — APPROVED** [review]

**Pull request merged**